### ISSUE TITLE
Wrong file path. File not found on build, but is present in project

### ISSUE THREAD

**Issue by Xarkam** [OmniSharp/omnisharp-vscode#1963]
## Environment data
`dotnet --info` output:
```
Outils en ligne de commande .NET (2.1.4)

Product Information:
 Version:            2.1.4
 Commit SHA-1 hash:  5e8add2190

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.16299
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.4\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.5
  Build    : 17373eb129b3b05aa18ece963f8795d65ef8ea54
```
VS Code version:
```
Version 1.19.2
Commit 490ef761b76b3f3b3832eff7a588aac891e5fe80
Date 2018-01-10T15:55:03.538Z
Shell 1.7.9
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64
```
C# Extension version:1.14.0-beta2
---
I have an error on a file not found when I build.
The path indicated in the error message does not seem to be the correct one.
![image](https://user-images.githubusercontent.com/331816/34995169-f7925cfa-fad5-11e7-9ed1-3afa484973fa.png)
(This is a french message. In english is file not found)
When I click on create a file, he tells me that it is impossible.
![image](https://user-images.githubusercontent.com/331816/34995198-05d6b216-fad6-11e7-8f2f-b1acdc84176a.png)
I have an unused variable in the file and as soon as I put it in comment, I have no problem.
![image](https://user-images.githubusercontent.com/331816/34995203-119ccbbc-fad6-11e7-8406-cae558d5ca73.png)

It's strange. When I put the variable in comment, I no longer have a file error not found.
When I run the dotnet build command line I have no error or warning.

You can try with my project located here: https://github.com/SOFTINUX/Base/tree/feature/security_extension
The file with unused variable `ExtensionDatabaseMetadata.cs` is in Security project.

I have this bug since already 2 version of omnisharp it seems to me

**Comment by rchande** [comment]
Thanks for the repro project @Xarkam--we will take a look.

**Comment by dsarfati** [comment]
I had a similar problem and found my solution in this duplicate? #2073. I added `/property:GenerateFullPaths=true` to my build args in my tasks.json files. Hope this helps.

**Comment by DustinCampbell** [comment]
This is a duplicate of https://github.com/OmniSharp/omnisharp-vscode/issues/1197. You can workaround the problem by updating your tasks.json file as @dsarfati mentioned.

**Comment by Xarkam** [comment]
I just realized that I failed to specify that my build is done by a .bat file
Because we can not chain tasks in the launch.json file.
Using a build file, the error persists.

And yes I have the argument on the build line in my .bat file
`dotnet build /property:GenerateFullPaths=true`

I have to do this because I have to copy dll for the application to work.

The day the launch.json file will execute tasks successively, then the workaround will work in my case.
